continuous_integration/envs/37-latest.yaml

```
@@ -14,3 +14,4 @@ dependencies:
   - pytest
   - pytest-cov
   - hilbertcurve
+  - pymorton
```

```suggestion
  - pip
  - pip:
    - pymorton
```

`pymorton` seems to be available only on PyPI. You'll need to do the same change in all envs.

---

updated & failed in 937bbbf (39-latest.yml) ... & 2c90399. I tried adding pymorton explicitly but not sure why it's failing?